src/python/library/tritonclient/http/__init__.py

```diff
@@ -1070,6 +1073,15 @@ def infer(self,
         query_params: dict
             Optional url query parameters to use in network
             transaction.
+        input_compression_algorithm : str
```

Comment:
request_compression_algorithm - "Optional HTTP compression algorithm to use for the request body..."

Same change for C++ API.